test-framework/src/test/java/org/apache/brooklyn/test/framework/TestSshCommandTest.java

```diff
@@ -278,6 +274,28 @@ public void shouldFailIfTestEntityHasNoMachine() throws Exception {
     }
     
     @Test
+    public void shouldFailFastIfNoCommand() throws Exception {
+        Map<String, ?> equalsZero = ImmutableMap.of(EQUALS, 0);
+        
+        TestSshCommand test = app.createAndManageChild(EntitySpec.create(TestSshCommand.class)
+                .configure(TIMEOUT, Duration.ONE_MINUTE)
```

Comment:
We've seen on apache jenkins that tests some times run extremely slowly (possibly due to thread starvation). I'm not sure we've ever reached 10 seconds, but certainly several seconds. Our general default timeout is 30 seconds in `Asserts.DEFAULT_LONG_TIMEOUT`, given these are not performance tests.

The reason I put it as one minute here is that if the fail-fast doesn't happen, then the existing default is 30 seconds. I don't want us to pass the test with a false-positive when it was really just a slow timeout, and the defaults being applied. But I also feel like leaving the "normal" default time in the assertion a few lines lower down. I could/should have referenced that constant rather than re-declaring 30 seconds again.